<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorStackedWindow</class>
 <widget class="QMainWindow" name="OperatorStackedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1618</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Модуль оператора</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Список конфигураций</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btmAllCams">
        <property name="text">
         <string>Все камеры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLog">
        <property name="minimumSize">
         <size>
          <width>103</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Журнал</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidgetConfigNames">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="tableWidgetVideoLabels"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QWidget" name="page2Widget" native="true">
            <property name="minimumSize">
             <size>
              <width>1200</width>
              <height>800</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1200</width>
              <height>800</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1618</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="dbSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Камеры</string>
    </property>
    <addaction name="turnOnAllCams"/>
    <addaction name="stopAllCams"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Зоны</string>
    </property>
    <addaction name="btnShowZoneOn"/>
    <addaction name="btnShowZoneOff"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Обнаружение объектов</string>
    </property>
    <addaction name="btnDetectObjectsOn"/>
    <addaction name="btnDetectObjectsOff"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="dbSettings">
   <property name="text">
    <string>База данных</string>
   </property>
  </action>
  <action name="turnOnAllCams">
   <property name="text">
    <string>Запуск</string>
   </property>
  </action>
  <action name="stopAllCams">
   <property name="text">
    <string>Остановка</string>
   </property>
  </action>
  <action name="btnShowZoneOn">
   <property name="text">
    <string>Отобразить</string>
   </property>
  </action>
  <action name="btnShowZoneOff">
   <property name="text">
    <string>Убрать</string>
   </property>
  </action>
  <action name="btnDetectObjectsOn">
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="btnDetectObjectsOff">
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
